tensorflow_privacy/privacy/membership_inference_attack/membership_inference_attack.py

```diff
@@ -170,6 +171,48 @@ def run_attacks(attack_input: AttackInputData,
       privacy_report_metadata=privacy_report_metadata)
 
 
+def run_seq2seq_attack(attack_input: Seq2SeqAttackInputData,
+                       privacy_report_metadata: PrivacyReportMetadata = None,
+                       balance_attacker_training: bool = True) -> AttackResults:
+  """Runs membership inference attacks on a seq2seq model.
+
+  Args:
+    attack_input: input data for running an attack
+    privacy_report_metadata: the metadata of the model under attack.
+    balance_attacker_training: Whether the training and test sets for the
+          membership inference attacker should have a balanced (roughly equal)
+          number of samples from the training and test sets used to develop
+          the model under attack.
+
+  Returns:
+    the attack result.
+  """
+  attack_input.validate()
+  attacker = models.LogisticRegressionAttacker()
```

Comment:
Copying this comment: "Can we add a comment why we don't support LR but not the other attackers?"

Reply:
Currently the seq2seq attack has a single number (average rank) in each record of the attack datasets, so I've used the simplest model i.e. LR to find the optimal threshold. 

Should we add support for the other attackers (MLP, RF, KNN) for single number features?

Reply:
My 50c: I think it's OK to keep linear regression, since it makes most sense for the single number features. I'd vote for dropping a comment explaining that for posterity, e.g.:

# Only linear regressions are supported, since they make most sense for single-number features